DQM/SiPixelMonitorCluster/src/SiPixelClusterSource.cc

```diff
@@ -465,9 +465,9 @@ void SiPixelClusterSource::getrococcupancy(DetId detId,const edm::DetSetVector<P
     if (modsign<0) { rocx = rocx -0.0001; }
     if (ladsign<0) { rocy = rocy -0.0001; } else { rocy = rocy +0.0001; }
     if (abs(DBladder)==1) { rocy = rocy + ladsign*0.5; } //take care of the half module
-    if (DBlayer==1) { meinput.at(0)->Fill(rocx,rocy); }
-    if (DBlayer==2) { meinput.at(1)->Fill(rocx,rocy); }
-    if (DBlayer==3) { meinput.at(2)->Fill(rocx,rocy); }
+    if (DBlayer==1) { meinput[DBlayer-1]->Fill(rocx,rocy); }
```

Comment:
@gouskos - actually the main point was to avoid the 3 if statements.. (if this code is running for phase 1, I guess you can not avoid 1 if statement, but presumably its not running there)